bundles/binding/org.openhab.binding.lcn/pom.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns="http://maven.apache.org/POM/4.0.0"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+	<parent>
+		<groupId>org.openhab.bundles</groupId>
+		<artifactId>binding</artifactId>
+		<version>1.8.0-SNAPSHOT</version>
+	</parent>
+
+	<name>openHAB LCN Binding</name>
+
+	<properties>
+		<bundle.symbolicName>org.openhab.binding.lcn</bundle.symbolicName>
+		<bundle.namespace>org.openhab.binding.lcn</bundle.namespace>
+		<deb.name>openhab-addon-binding-lcn</deb.name>
+		<deb.description>${project.name}</deb.description>
+	</properties>
+
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>org.openhab.binding</groupId>
+	<artifactId>org.openhab.binding.lcn</artifactId>
+
+	<packaging>eclipse-plugin</packaging>
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.vafer</groupId>
+				<artifactId>jdeb</artifactId>
+			</plugin>
+		</plugins>
+	</build>
```

Comment:
add compiler directive to compile with 1.7 source level (see pom.xml of XMPP Action) since the default is 1.6